#hipnuc config
imu_serial: "/dev/ttyUSB1"
baud_rate: 460800
imu_axes: "ENU"          #hipnuc imu default:"ENU"   ros carrier system:"FLU"
frame_id: "imu_link"

imu_topic: "/imu/data"
imu_enable: true

mag_topic: "/MAG_data"
mag_enable: false

eul_topic: "/EUL_data"
eul_enable: false

pre_topic: "/PRE_data"
pre_enable: false

temp_topic: "/TEMP_data"
temp_enable: false

port_timeout_ms: 500


